drivers/storage/portworx/preflight.go

```diff
@@ -295,24 +295,43 @@ func (u *preFlightPortworx) RunPreFlight() error {
 func (u *preFlightPortworx) ProcessPreFlightResults(recorder record.EventRecorder, storageNodes []*corev1.StorageNode) error {
 	logrus.Infof("pre-flight: process pre-flight results...")
 
-	passed := true
-	for _, node := range storageNodes {
-		logrus.Infof("storageNode[%s]: %#v ", node.Name, node.Status.Checks)
-		if len(node.Status.Checks) > 1 {
-			for _, check := range node.Status.Checks {
-				if check.Type != "status" {
-					msg := fmt.Sprintf("%s pre-flight check ", check.Type)
-					if check.Success {
-						msg = msg + "passed: " + check.Reason
-						k8sutil.InfoEvent(recorder, u.cluster, util.PassPreFlight, msg)
+	passed := false
+	if len(storageNodes) > 0 {
```

Comment:
Shall we handle `len(storageNodes) = 0` case as an error? 
The current code would return nil with no action taken, wonder if this is the desired behavior.